pkg/query-service/constants/constants.go

```diff
@@ -236,6 +236,14 @@ const (
 		"CAST((attributes_int64_key, attributes_int64_value), 'Map(String, Int64)') as  attributes_int64," +
 		"CAST((attributes_float64_key, attributes_float64_value), 'Map(String, Float64)') as  attributes_float64," +
 		"CAST((resources_string_key, resources_string_value), 'Map(String, String)') as resources_string "
+	TracesExplorerViewSQLSelectWithSubQuery = "WITH subQuery AS ( " +
+		"SELECT distinct on (traceID) traceID, durationNano, serviceName, name " +
+		"FROM " + SIGNOZ_TRACE_DBNAME + "." + SIGNOZ_SPAN_INDEX_TABLENAME +
+		" WHERE " + "%s " + "%s " + "ORDER BY durationNano DESC "
+	TracesExplorerViewSQLSelectQuery = "SELECT subQuery.serviceName, subQuery.name, count() AS span_count, subQuery.durationNano, " +
+		"traceID FROM " + SIGNOZ_TRACE_DBNAME + "." + SIGNOZ_SPAN_INDEX_TABLENAME + " INNER JOIN subQuery ON " + SIGNOZ_SPAN_INDEX_TABLENAME +
+		".traceID = subQuery.traceID GROUP BY traceID, subQuery.durationNano, subQuery.name, subQuery.serviceName " +
+		"ORDER BY subQuery.durationNano desc;"
```

Comment:
Format with backticks, it will be clean.

Reply:
I made it cleaner by using formatting. Using backticks makes tests fail maybe due to multiline char.